source/jormungandr/readme.md

```diff
@@ -131,4 +131,20 @@ $ workon jormung
 
 ### sqlalchemy.exc.OperationalError
 
-Jormungandr is trying to access a "jormungandr" database. You can disable this default behavior with `JORMUNGANDR_DISABLE_DATABASE=True`.
\ No newline at end of file
+Jormungandr is trying to access a "jormungandr" database. You can disable this default behavior with `JORMUNGANDR_DISABLE_DATABASE=True`.
+
+# How to profile Jormungandr? 
+
+The profiling of Jormungandr is done thanks to Werkzeug profiler, for more details, one can read this interesting [article](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xvi-debugging-testing-and-profiling), more specifically, in the Profiling section.
+
+To enable the profiling of Jormungandr, one must set `ACTIVATE_PROFILING` to True in the `$JORMUNGANDR_CONFIG_FILE`[default_settings.py](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/default_settings.py).
+Then set the correct directory(create one if not exists) for the output of profiler [profile](https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/api.py#L130).
+
+The profiling is triggered on every arrival of a request, so a profile file is generated when a request is processed.
+
+Once the profile file is generated, download `gprof2dot` with `pip install gprof2dot` and run the following command to visualize the graph `gprof2dot -f pstats your_profiling.prof | dot -Tsvg -o callgraph.svg`
+
```

Comment:
Add the fact that the callgraph file is generated and it needs to be opened manually